drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -738,3 +738,52 @@ int stm32_i2c_configure_timing(const struct device *dev, uint32_t clock)
 
 	return 0;
 }
+
+int stm32_i2c_transaction(const struct device *dev,
+		      struct i2c_msg msg, uint8_t *next_msg_flags,
+		      uint16_t periph)
+{
+	/*
+	 * Perform a I2C transaction, while taking into account the STM32 I2C
```

Comment:
To make sure people understand this differenciation was made on purpose, I propose to make it clear:
```suggestion
	 * Perform a I2C transaction, while taking into account the STM32 I2C V2
```